src/workloads/sharding/UpdateOneWithoutShardKey.yml

```diff
@@ -0,0 +1,151 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  On a single shard cluster, compare the latency of single write commands without shard key on a sharded collection vs unsharded collection.
```

Comment:
I think this description should be of what the workload is doing, not really what the meta goal is. That we can put in the ticket or something.

Reply:
Also we can technically run multiple updateOnes in a single batch.

Reply:
Done. I think it's better to run as single update commands to compare latency linearly, so keeping thread size as 1.

Reply:
Oh I meant like multiple writes in a batch, not multiple threads. Here is some documentation on the CrudActor: https://github.com/mongodb/genny/blob/master/src/workloads/docs/CrudActor.yml